integreat_cms/cms/templates/pois/poi_form.html

```diff
@@ -167,6 +167,9 @@ <h3 class="font-bold text-black heading">
                 <div id="right-sidebar-column"
                      class="flex flex-col flex-wrap 3xl:col-end-3 4xl:col-end-auto md:w-full">
                     {% include "./poi_form_sidebar/contact_box.html" with box_id="poi-contact" %}
+                    {% if not poi_form.instance.id and perms.cms.change_contact %}
```

Comment:
```suggestion
                    {% if poi_form.instance.id and perms.cms.change_contact %}
```
I guess you want to show this box only when an existing POI is being edited, not when a new POI is being created on the POI form.